python/entry_points/py_console_script_binary.bzl

```diff
@@ -13,12 +13,14 @@
 # limitations under the License.
 
 """
-Creates an executable (a non-test binary) for console_script entry points.
-
-```{include} /_includes/py_console_script_binary.md
-```
+Please use `py_console_script_binary` from `//python:pip_bzl` instead.
 """
 
-load("//python/private:py_console_script_binary.bzl", _py_console_script_binary = "py_console_script_binary")
+load("//python:pip.bzl", _py_console_script_binary = "py_console_script_binary")
+
+def _compatibility_shim(**kwargs):
+    # buildifier: disable=print
+    print("Please use `py_console_script_binary` from `//python:pip_bzl` instead.")
```

Comment:
I saw people using deprecation attribute in some cases instead of a print (this is in py_proto_library case). That may be less spammy, as this will cause one print invocation per usage of the rule, I think?

Maybe it would be useful to print a buildozer command for replacing the loads?

Reply:
SGTM. Didn't know about it. Will FIXUP.

Reply:
For macros, I think there's a doc string syntax that stardoc parses out into a "deprecation" doc text. I think it's either `"""DEPRECATED: blabla """` or `"""[DEPRECATED]: blalba"""` ? Not entirely sure.

I'm pretty sure our docgen respects it when stardoc emits it.